src/writer.rs

```diff
@@ -1436,6 +1436,22 @@ mod tests {
         assert_eq!(wtr_as_string(wtr), "42,42.5,true\n");
     }
 
+    #[test]
+    fn serialize_no_headers_128() {
+        #[derive(Serialize)]
+        struct Row {
+            foo: i128,
+            bar: f64,
+            baz: bool,
+        }
+
+        let mut wtr = WriterBuilder::new()
+            .has_headers(false)
+            .from_writer(vec![]);
+        wtr.serialize(Row { foo: 9_223_372_036_854_775_808, bar: 42.5, baz: true }).unwrap();
```

Comment:
This line still looks longer than 79 columns.

Reply:
I missed that one, sorry. Should be fixed now.